hardware.go

```diff
@@ -8,6 +8,15 @@ import (
 	"strings"
 )
 
+// alert can be a buzzer or a light, or something else ...
+func activateAlert() error {
+	_, err = exec.Command("python", "hardware/alert.py").Output()
```

Comment:
Now that we have a functioning `config`, why not add the types of alerts that are enabled into it, and split them out into multiple files?
That way, we could add `buzz` and `print` alert types, and just iterate through the enabled ones here.

It'd allow for easy extension to other alert types too, like `email` or even `sms` :D